### DISCUSSION TITLE
Can we add Height equal width ?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#2779] [Ideas]
Hi, I have an image like this
`<img src=".." class="w-1/8 " />
`can we have a class like this? `<img src=".." class="w-1/8 h-eqw" />` h-eqw = height = width
Just an idea to consider. Thank you. I love Tailwind <3

**Comment by ThaddeusJiang** [comment]
good idea

**Comment by MathGeniusJodie** [comment]
what would the generated css look like? this isn't exactly an easy problem to solve

**Reply by IRediTOTO** [reply]
:)) https://css-tricks.com/a-couple-of-use-cases-for-calc/
Here is it, but i dont think it work.

**Comment by samurai-techlead** [comment]
> Hi, I have an image like this `<img src=".." class="w-1/8 " /> `can we have a class like this? `<img src=".." class="w-1/8 h-eqw" />` h-eqw = height = width Just an idea to consider. Thank you. I love Tailwind <3

It's good idea, but here is any update?

**Reply by alex-krasikau** [reply]
The `size` utility class will be included in the next release: https://github.com/tailwindlabs/tailwindcss/pull/12287

**Comment by CodeAunt** [comment]
`aspect-square` works for me

**Reply by samurai-techlead** [reply]
Perfect!

**Reply by huanghanzhilian** [reply]
Thank you for your solutions and ideas, which helped me solve problems on an open source project: https://github.com/huanghanzhilian/c-shopping-rn

**Comment by MURHAF-ELMASRI** [comment]
To make the height the same as the width in Tailwind, you can use the ```aspect-square``` class. This will ensure the element maintains a 1:1 aspect ratio.
